src/help/guides/adding-repos.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Adding Repositories
+lang: en-US
+---
+
+# External Source Repositories
+## What is an External Repository?
+**Paperback** out of the box only comes with **MangaDex** as a source for manga. The app however has support for adding manga from external repositories which are not on the official listings. Beta Sources, 18+ Sources, and sources developed independently are seperated due to **Paperback** having to adhere to Apple policies in order to be distributed through TestFlight and the Apple App Store. 
```

Comment:
I personally find this section a bit confuse.

We say `**Paperback** out of the box only comes with **MangaDex** as a source for manga` in the `What is an External Repository?` Moreover I believe the Official Repository is also built into Paperback. I'm not sure about the installation process but here is what I did understood: Paperback come with MangaDex and users can add sources from the official repo. Then there is also a support for external repo.

It is also possible to add repo that are on the official listing like H-extensions.

---

Shouldn't this be `source`?
```suggestion
**Paperback** out of the box only comes with **MangaDex** as a source for manga. The app however has support for adding sources from external repositories which are not on the official listings. Beta Sources, 18+ Sources, and sources developed independently are seperated due to **Paperback** having to adhere to Apple policies in order to be distributed through TestFlight and the Apple App Store. 
```

---

I believe the issue in the section is the confusion of `the mangas from a source`, the `source` and the `source repo`.
An idea could be to explain first that other sources are supported and then that external repo can be added to add more sources.